<!doctype html>
<html>
    <head>
        <meta charset="utf-8" />
        <meta name="viewport" content="width=device-width, initial-scale=1" />
        <title>Plant-Based LLMs</title>
        <style type="text/css">
            html {
                font-family: sans-serif;
            }
            table {
                border: 1px solid #aaa;
            }
            tr:nth-child(odd) {
                background-color: #f0f0f0;
            }
            tr:nth-child(odd) td {
                border-bottom: 1px solid #aaa;
            }
            th,
            td {
                padding: 4px;
            }
            a {
                cursor: pointer;
                text-decoration: none;
            }
            a:hover {
                text-decoration: underline;
            }
            .bracket {
              border: 1px solid #000;
              border-radius: 10px;
              margin-bottom: 15px;
              padding: 10px;
            }
        </style>
    </head>
    <body>
      <p>
        LLMs trained on plant genomes and proteomes.
        For context or questions, see
        <a href="https://github.com/mapmeld/plant-based-llms">the GitHub repo</a
        >.
      </p>

      <p>
        Skip to: <a href="#plant">Plant-Based</a> | <a href="#inclusive">Plant-Inclusive</a> | <a href="#other">Other Notable</a>
      </p>

      <a name="plant"></a>
      <div class="bracket">
        <h4>Plant-Based LLMs / Research</h4>

        <table>
            <tr>
                <th>Name</th>
                <th>Source</th>
                <th>Tokenization</th>
                <th>HF Link</th>
                <th>GH Link</th>
            </tr>
            <tr>
                <td>AgroNT</td>
                <td>InstaDeep</td>
                <td>6bp/token, 1000x window</td>
                <td>
                    <a
                        href="https://huggingface.co/InstaDeepAI/agro-nucleotide-transformer-1b"
                    >
                        InstaDeepAI/agro-nucleotide-transformer-1b
                    </a>
                </td>
                <td>
                    <a
                        href="https://github.com/instadeepai/nucleotide-transformer"
                    >
                        instadeepai/nucleotide-transformer
                    </a>
                </td>
            </tr>
            <tr>
                <td colspan="5">
                    Inference:
                    https://github.com/instadeepai/nucleotide-transformer/blob/main/examples/inference.ipynb
                    Inference on segment:
                    https://github.com/instadeepai/nucleotide-transformer/blob/main/examples/inference_segment_nt.ipynb
                    For non-plant DNA, see:
                    https://huggingface.co/InstaDeepAI/nucleotide-transformer-2.5b-multi-species
                </td>
            </tr>

            <tr>
                <td>PlantCaduceus</td>
                <td>
                    <a href="https://kuleshov-group.github.io/"
                        >Kuleshov Group</a
                    >, Cornell University, USA
                </td>
                <td>1bp/token, 512 tokens x bidirectional</td>
                <td>
                    <a
                        href="https://huggingface.co/kuleshov-group/PlantCaduceus_l32"
                    >
                        kuleshov-group/PlantCaduceus_l32
                    </a>
                </td>
                <td>
                    <a href="https://github.com/kuleshov-group/PlantCaduceus">
                        kuleshov-group/PlantCaduceus
                    </a>
                </td>
            </tr>
            <tr>
                <td colspan="5">
                    Basics:
                    https://github.com/kuleshov-group/PlantCaduceus/blob/main/notebooks/examples.ipynb
                </td>
            </tr>

            <tr>
                <td>Plant-*-BPE</td>
                <td>
                    <a href="https://zhangtaolab.org/">Zhang Tao Lab</a>,
                    Yangzhou University, China
                </td>
                <td>nucleotide / BPE</td>
                <td>
                    <a
                        href="https://huggingface.co/collections/zhangtaolab/plant-foundation-models-668514bd5c384b679f9bdf17"
                    >
                        collections/zhangtaolab/plant-foundation-models
                    </a>
                    (multiple architectures: Mamba, BERT, GPT)
                </td>
                <td>
                    <a href="https://github.com/zhangtaolab/plant_DNA_LLMs">
                        zhangtaolab/plant_DNA_LLMs
                    </a>
                </td>
            </tr>
            <tr>
                <td colspan="5">
                    finetune:
                    https://github.com/zhangtaolab/Plant_DNA_LLMs/blob/main/scripts/run_finetune_test.sh
                    inference:
                    https://github.com/zhangtaolab/Plant_DNA_LLMs/blob/main/scripts/run_inference_test.sh
                </td>
            </tr>

            <tr>
                <td>PlantRNA-FM</td>
                <td>
                  <a href="https://colalab.ai/">ColaLAB</a> @ University of Exeter and the
                  <a href="https://www.jic.ac.uk/about-us/">John Innes Centre</a> @ Norwich Research Park, UK</td>
                <td>EsmTokenizer</td>
                <td>
                    <a href="https://huggingface.co/yangheng/PlantRNA-FM">
                        yangheng/PlantRNA-FM</a
                    >
                </td>
                <td>
                    <a href="https://github.com/yangheng95/PlantRNA-FM"
                        >yangheng95/PlantRNA-FM</a
                    >
                </td>
            </tr>
            <tr>
                <td colspan="5">
                    region classification
                    https://github.com/yangheng95/PlantRNA-FM/blob/master/rice_region_classification.py
                    also see https://huggingface.co/yangheng/OmniGenome-52M and
                    https://github.com/yangheng95/OmniGenBench
                </td>
            </tr>

            <tr>
                <td>PlantBERT</td>
                <td>University of Potsdam, Germany</td>
                <td>multiple nucleotides</td>
                <td>
                    <a href="https://huggingface.co/nigelhartm/PlantBERT"
                        >nigelhartm/PlantBERT</a
                    >
                </td>
                <td>
                    <a href="https://github.com/nigelhartm/PlantBERT"
                        >nigelhartm/PlantBERT</a
                    >
                </td>
            </tr>
            <tr>
                <td colspan="5">
                    finetune:
                    https://github.com/nigelhartm/PlantBERT/blob/main/scripts/08_finetune_all_gue.py
                </td>
            </tr>

            <tr>
                <td>FloraBERT</td>
                <td>
                    Institute for Applied Computational Sciences, Harvard
                    University, USA
                </td>
                <td>multiple nucleotides</td>
                <td>
                    <a href="https://huggingface.co/Gurveer05/FloraBERT">
                        Gurveer05/FloraBERT
                    </a>
                </td>
                <td>
                    <a href="https://github.com/gurveervirk/florabert"
                        >gurveervirk/florabert</a
                    >
                </td>
            </tr>
            <tr>
                <td colspan="5">
                    finetune
                    https://github.com/benlevyx/florabert/blob/main/scripts/1-modeling/finetune.py
                </td>
            </tr>

            <tr>
                <td>Farmer.chat</td>
                <td>Digital Green and CGIAR, India</td>
                <td>natural language LLMs</td>
                <td>
                    <a href="https://huggingface.co/blog/digital-green-llm-judge">HF blog</a>
                </td>
                <td>
                    <a href="https://github.com/GooeyAI">GooeyAI</a>
                </td>
            </tr>
            <tr>
                <td colspan="5">
                </td>
            </tr>

            <tr>
                <td>protein-matryoshka-embeddings</td>
                <td>it me, USA</td>
                <td>amino acids</td>
                <td>
                    <a
                        href="https://huggingface.co/monsoon-nlp/protein-matryoshka-embeddings"
                        >monsoon-nlp/protein-matryoshka-embeddings</a
                    >
                </td>
                <td>--</td>
            </tr>
            <tr>
                <td colspan="5">
                    dataset of pairs
                    https://huggingface.co/datasets/monsoon-nlp/protein-pairs-uniprot-swissprot <br/>
                    plant protein and text descriptions: https://huggingface.co/datasets/monsoon-nlp/greenbeing-proteins <br/>
                    plant protein descriptions converted to binary classification tasks: https://huggingface.co/datasets/monsoon-nlp/greenbeing-binary
                </td>
            </tr>
        </table>
      </div>

        <a name="inclusive"></a>
        <div class="bracket">
        <h4>"Plant-Inclusive" LLMs / Research</h4>

        Trained on cells and proteins in general, including plants in training.

        <table>
            <tr>
              <th>Name</th>
              <th>Source</th>
              <th>Tokenization</th>
              <th>HF Link</th>
              <th>GH Link</th>
            </tr>
            <tr>
                <td>gLM2</td>
                <td>
                    <a href="https://www.tatta.bio/">Tatta Bio</a>
                </td>
                <td>hybrid 1bp or 1 amino acid/token, 4096x window</td>
                <td>
                    <a href="https://huggingface.co/tattabio/gLM2_650M">
                        tattabio/gLM2_650M
                    </a>
                </td>
                <td>
                    <a href="https://github.com/TattaBio/gLM2">
                        TattaBio/gLM2
                    </a>
                </td>
            </tr>
            <tr>
                <td colspan="5">
                    Example:
                    https://github.com/TattaBio/gLM2/blob/main/categorical_jacobian_gLM2.ipynb
                  Embeddings: https://huggingface.co/tattabio/gLM2_650M_embed 
                </td>
            </tr>

            <tr>
                <td>ProtNote</td>
                <td>Microsoft</td>
                <td>amino acids + annotation codes</td>
                <td>--</td>
                <td>
                    <a href="https://github.com/microsoft/protnote">
                        microsoft/protnote
                    </a>
                </td>
            </tr>
            <tr>
                <td colspan="5">
                    Notebooks:
                    https://github.com/microsoft/protnote/tree/main/notebooks
                </td>
            </tr>

            <tr>
                <td>ESM-2</td>
                <td>Facebook / Meta</td>
                <td>ESMTokenizer / amino acids</td>
                <td>
                  <a href="https://huggingface.co/facebook/esm2_t48_15B_UR50D">
                    facebook/esm2_t48_15B_UR50D
                  </a>
                </td>
                <td>
                    <a href="https://github.com/facebookresearch/esm">
                        facebookresearch/esm
                    </a> (archived)
                </td>
            </tr>
            <tr>
                <td colspan="5">
                    Many finetuned models on HF, such as https://huggingface.co/blog/AmelieSchreiber/esmbind
                    <br/ >
                    With "Flash Attention" https://github.com/pengzhangzhi/faesm
                </td>
            </tr>

            <tr>
                <td>Bio Embeddings</td>
                <td>Technical University of Munich, Germany</td>
                <td>amino acids (?)</td>
                <td>supports multiple models</td>
                <td>
                    <a href="https://github.com/sacdallago/bio_embeddings"
                        >sacdallago/bio_embeddings</a
                    >
                </td>
            </tr>
            <tr>
              <td colspan="5">
                http://data.bioembeddings.com/
              </td>
            </tr>

            <tr>
                <td>ProtTrans</td>
                <td>multiple</td>
                <td>amino acids</td>
                <td><a href="https://huggingface.co/Rostlab">Rostlab</a></td>
                <td>
                    <a href="https://github.com/agemagician/ProtTrans"
                        >agemagician/ProtTrans</a
                    >
                </td>
            </tr>
            <tr></tr>

            <tr>
                <td>GPN</td>
                <td>
                    <a href="https://people.eecs.berkeley.edu/~yss/group.html"
                        >Song Lab</a
                    >, U.C. Berkeley, USA
                </td>
                <td>nucleotide</td>
                <td>
                    <a
                        href="https://huggingface.co/collections/songlab/gpn-653191edcb0270ed05ad2c3e"
                    >
                        collections/songlab/gpn
                    </a>
                    (model, tokenizer, dataset)
                </td>
                <td>
                    <a href="https://github.com/songlab-cal/gpn">
                        songlab-cal/gpn
                    </a>
                </td>
            </tr>
            <tr>
                <td colspan="5">
                    Embedding:
                    https://github.com/songlab-cal/gpn/blob/main/examples/ss/basic_example.ipynb
                    Multiple Sequence Alignment (MSA):
                    https://github.com/songlab-cal/gpn/blob/main/examples/msa/basic_example.ipynb
                </td>
            </tr>

            <tr>
              <td>BioM3</td>
              <td>U. Chicago, USA</td>
              <td>amino acid, protein design from natural text</td>
              <td>
                --
              </td>
              <td>
                --
              </td>
            </tr>
            <tr>
              <td colspan="5">
                Paper: https://www.biorxiv.org/content/10.1101/2024.11.11.622734
              </td>
            </tr>

            <tr>
                <td>ProteinBERT</td>
                <td>Hebrew University of Jerusalem, Israel</td>
                <td>1/amino acid</td>
                <td>
                    <a href="https://huggingface.co/GrimSqueaker/proteinBERT"
                        >GrimSqueaker/proteinBERT</a
                    >
                </td>
                <td>
                    <a href="https://github.com/nadavbra/protein_bert">
                        nadavbra/protein_bert
                    </a>
                </td>
            </tr>
            <tr>
                <td colspan="5">
                    demo notebook
                    https://github.com/nadavbra/protein_bert/blob/master/ProteinBERT%20demo.ipynb
                </td>
            </tr>

            <tr>
                <td>Mistral-DNA</td>
                <td>Paul Sabatier University, France</td>
                <td>nucleotide</td>
                <td>
                    <a
                        href="https://huggingface.co/RaphaelMourad/Mistral-DNA-v1-417M-Athaliana"
                    >
                        RaphaelMourad/Mistral-DNA-v1-417M-Athaliana
                    </a>
                </td>
                <td>
                    <a href="https://github.com/raphaelmourad/Mistral-DNA"
                        >raphaelmourad/Mistral-DNA</a
                    >
                </td>
            </tr>
            <tr>
                <td colspan="5">
                    finetune
                    https://github.com/raphaelmourad/Mistral-DNA/blob/main/scriptPython/script_finetune.ipynb
                </td>
            </tr>

            <tr>
                <td>PAIR</td>
                <td>Vector Institute / U. Toronto, Canada</td>
                <td>amino acid</td>
                <td>
                    <a href="https://huggingface.co/h4duan"> h4duan </a>
                    (finetunes of ESM, ProtT5)
                </td>
                <td>--</td>
            </tr>
            <tr>
                <td colspan="5">
                    Sample use: https://huggingface.co/h4duan/PAIR-prott5
                </td>
            </tr>

            <tr>
                <td>ProtBFN</td>
                <td>InstaDeep</td>
                <td>amino acids</td>
                <td>
                    <a
                        href="https://huggingface.co/InstaDeepAI/protein-sequence-bfn"
                    >
                        InstaDeepAI/protein-sequence-bfn
                    </a>
                </td>
                <td>
                    <a
                        href="https://github.com/instadeepai/protein-sequence-bfn"
                    >
                        instadeepai/protein-sequence-bfn
                    </a>
                </td>
            </tr>
            <tr>
                <td colspan="5">
                    Sample
                    https://github.com/instadeepai/protein-sequence-bfn/blob/main/sample.py
                </td>
            </tr>

            <tr>
                <td>CHEAP Embeddings</td>
                <td>multiple, USA</td>
                <td>amino acids</td>
                <td>
                    <a href="https://huggingface.co/amyxlu/cheap-proteins"
                        >amyxlu/cheap-proteins</a
                    >, based on
                    <a href="https://huggingface.co/facebook/esmfold_v1"
                        >ESMFold</a
                    >
                </td>
                <td>
                    <a href="https://github.com/amyxlu/cheap-proteins">
                        amyxlu/cheap-proteins</a
                    >
                </td>
            </tr>
            <tr></tr>

            <tr>
                <td>ProTrek</td>
                <td>
                    Westlake Representation Learning Lab, Westlake University,
                    China
                </td>
                <td>EsmTokenizer</td>
                <td>
                    <a
                        href="https://huggingface.co/westlake-repl/ProTrek_650M_UniRef50"
                    >
                        westlake-repl/ProTrek_650M_UniRef50
                    </a>
                    and see
                    <a
                        href="https://huggingface.co/westlake-repl/SaProt_650M_AF2"
                        >SaProt_650M_AF2</a
                    >
                </td>
                <td>
                    <a href="https://github.com/westlake-repl/ProTrek"
                        >westlake-repl/ProTrek</a
                    >
                </td>
            </tr>
            <tr>
                <td colspan="5">
                    CoLab
                    https://colab.research.google.com/drive/1On2xQU0d7351bIBgZpz2T0VUp2gZium0?usp=sharing
                </td>
            </tr>

            <tr>
              <td>Prot-xLSTM</td>
              <td>JKU Linz, Austria</td>
              <td>amino acids</td>
              <td><a href="https://ml.jku.at/research/Bio-xLSTM/downloads/Prot-xLSTM/">ml.jku.at</a></td>
              <td><a href="https://github.com/ml-jku/Prot-xLSTM">ml-jku/Prot-xLSTM</a></td>
            </tr>
            <tr>
              <td colspan="5">
                evaluation: https://github.com/ml-jku/Prot-xLSTM/tree/main/evaluation
                generation: https://github.com/ml-jku/Prot-xLSTM/blob/main/examples/generation.ipynb
                variant fitness: https://github.com/ml-jku/Prot-xLSTM/blob/main/examples/variant_fitness.ipynb
              </td>
            </tr>

            <tr>
                <td>RNA-FM / RhoFold</td>
                <td>
                    multiple
                </td>
                <td>nucleotides</td>
                <td>
                    (<a href="https://drive.google.com/drive/folders/1VGye74GnNXbUMKx6QYYectZrY7G2pQ_J">official</a> on Google Drive)<br/>
                    (<a href="https://huggingface.co/multimolecule/rnafm">unofficial</a> on HF/multimolecule)
                </td>
                <td>
                    <a href="https://github.com/ml4bio/RNA-FM"
                        >ml4bio/RNA-FM</a
                    >
                </td>
            </tr>
            <tr></tr>

            <tr>
                <td>Protein-Vec</td>
                <td>
                    <a href="https://www.gleghornlab.com/">Gleghorn Lab</a>, U.
                    Delaware, USA
                </td>
                <td></td>
                <td>
                    <a href="https://huggingface.co/lhallee/ProteinVec"
                        >lhallee/ProteinVec</a
                    >
                </td>
                <td>
                    <a href="https://github.com/lhallee/ProteinVecHuggingface"
                        >lhallee/ProteinVecHuggingface</a
                    >
                </td>
            </tr>
            <tr></tr>

            <tr>
                <td>MAMMAL / biomed.omics / biomed-multi-alignment</td>
                <td>IBM, USA</td>
                <td>amino acids, gene names</td>
                <td>
                    <a
                        href="https://huggingface.co/ibm/biomed.omics.bl.sm.ma-ted-458m.protein_solubility"
                        >ibm/biomed.omics.bl.sm.ma-ted-458m.protein_solubility</a
                    >
                </td>
                <td>
                    <a
                        href="https://github.com/BiomedSciAI/biomed-multi-alignment"
                    >
                        BiomedSciAI/biomed-multi-alignment</a
                    >
                </td>
            </tr>
            <tr>
                <td colspan="5">
                    tutorial creating a new task
                    https://github.com/BiomedSciAI/biomed-multi-alignment/blob/main/tutorials/advanced_create_new_task.ipynb
                </td>
            </tr>

            <tr>
                <td>ProteinGLM</td>
                <td>Tsinghua University, China</td>
                <td>amino acids</td>
                <td>
                    <a href="https://huggingface.co/Bo1015/proteinglm-100b-int4"
                        >Bo1015/proteinglm-100b-int4</a
                    >
                </td>
                <td>
                    <a href="https://github.com/allanchen95/xTrimoPGLM"
                        >allanchen95/xTrimoPGLM</a
                    >
                </td>
            </tr>
            <tr>
                <td colspan="5">
                    contact prediction
                    https://github.com/allanchen95/xTrimoPGLM/blob/main/finetune/lora_contact_prediction.py
                </td>
            </tr>

            <tr>
                <td>PSALM</td>
                <td>Harvard University, USA</td>
                <td>ESMTokenizer</td>
                <td>
                    <a href="https://huggingface.co/ProteinSequenceAnnotation"
                        >ProteinSequenceAnnotation</a
                    >
                </td>
                <td>
                    <a href="https://github.com/Protein-Sequence-Annotation/PSALM"
                        >Protein-Sequence-Annotation/PSALM</a
                    >
                </td>
            </tr>
            <tr>
            </tr>

            <tr>
                <td>Protein-Llama-3-8B and Protein-Phi-3-mini</td>
                <td>Esperanto Technologies, USA</td>
                <td></td>
                <td>
                    <a
                        href="https://huggingface.co/collections/Esperanto/esperanto-fine-tuned-llm-based-models-673486672f0f0d8e0e14c985"
                    >
                        collections/Esperanto/esperanto-fine-tuned-llm-based-models
                    </a>
                </td>
                <td>--</td>
            </tr>
            <tr>
                <td colspan="5">--</td>
            </tr>

            <tr>
                <td>ProteinDT</td>
                <td>multiple</td>
                <td>amino acids</td>
                <td>
                    <a
                        href="https://huggingface.co/chao1224/ProteinDT/tree/main/ProteinDT"
                    >
                        chao1224/ProteinDT
                    </a>
                </td>
                <td>
                  <a href="https://github.com/chao1224/ProteinDT">chao1224/ProteinDT</a>
                </td>
            </tr>
            <tr>
                <td colspan="5">--</td>
            </tr>

            <tr>
                <td>SeqDance</td>
                <td><a href="http://www.columbia.edu/~ys2411/">Shen Lab</a>, Columbia University, USA</td>
                <td>EsmTokenizer</td>
                <td>
                    <a
                        href="https://zenodo.org/records/13909695"
                    >
                        on Zenodo
                    </a>
                </td>
                <td>
                  <a href="https://github.com/ShenLab/SeqDance">ShenLab/SeqDance</a>
                </td>
            </tr>
            <tr>
                <td colspan="5">--</td>
            </tr>
        </table>
      </div>

        <a name="other"></a>
        <div class="bracket">
        <h4>Other notable biology + LLM work</h4>
        <p>Not trained on any plants, but seems important to the bio research community. Includes datasets, tool-use</p>
        <table>
            <tr>
                <th>Name</th>
                <th>Source</th>
                <th>HF Link</th>
                <th>GH Link</th>
            </tr>
            <tr>
                <td>Evo</td>
                <td>
                    <a href="https://evodesign.org/"
                        >Laboratory of Evolutionary Design</a
                    >, Stanford University, USA
                </td>
                <td>
                    <a
                        href="https://huggingface.co/togethercomputer/evo-1-131k-base"
                    >
                        togethercomputer/evo-1-131k-base
                    </a>
                </td>
                <td>
                    <a href="https://github.com/evo-design/evo">
                        evo-design/evo
                    </a>
                    and
                    <a href="https://github.com/togethercomputer/stripedhyena"
                        >togethercomputer/stripedhyena</a
                    >
                </td>
            </tr>
            <tr>
                <td colspan="4">
                    Inference example:
                    https://github.com/evo-design/evo/blob/main/scripts/example_inference.py
                </td>
            </tr>

            <tr>
                <td>HyenaDNA</td>
                <td>
                    <a href="https://cs.stanford.edu/people/chrismre/"
                        >Hazy Research</a
                    >, Stanford University, USA
                </td>
                <td>
                    <a
                        href="https://huggingface.co/LongSafari/hyenadna-large-1m-seqlen"
                    >
                        LongSafari/hyenadna-large-1m-seqlen
                    </a>
                </td>
                <td>
                    <a href="https://github.com/HazyResearch/hyena-dna">
                        HazyResearch/hyena-dna
                    </a>
                </td>
            </tr>
            <tr>
                <td colspan="4">
                    Evaluation scripts:
                    https://github.com/HazyResearch/hyena-dna/tree/main/evals
                </td>
            </tr>

            <tr>
                <td>Open MetaGenomic Dataset OMG</td>
                <td>
                    <a href="https://www.tatta.bio/">Tatta Bio</a>
                </td>
                <td>
                    <a href="https://huggingface.co/datasets/tattabio/OMG"
                        >datasets/tattabio/OMG</a
                    >
                </td>
                <td>
                    <a href="https://github.com/TattaBio/OMG"> TattaBio/OMG </a>
                </td>
            </tr>
            <tr></tr>

            <tr>
                <td>scGPT</td>
                <td>
                    <a href="https://wanglab.ai/">WangLab</a>, U. Toronto,
                    Canada
                </td>
                <td>--</td>
                <td>
                    <a href="https://github.com/bowang-lab/scGPT">
                        bowang-lab/scGPT
                    </a>
                </td>
            </tr>
            <tr>
                <td colspan="4">
                    Finetune:
                    https://github.com/bowang-lab/scGPT/blob/main/examples/finetune_integration.py
                </td>
            </tr>

            <tr>
                <td>AllTheBacteria</td>
                <td>European Molecular Biology Laboratory, UK</td>
                <td>--</td>
                <td>
                    <a href="https://github.com/AllTheBacteria/AllTheBacteria">
                        AllTheBacteria/AllTheBacteria
                    </a>
                    dataset
                </td>
            </tr>
            <tr></tr>

            <tr>
                <td>Genomic Benchmarks</td>
                <td>CEITEC, Czech Republic</td>
                <td>
                    <a href="https://huggingface.co/katarinagresova"
                        >https://huggingface.co/katarinagresovakatarinagresova</a
                    >
                </td>
                <td>
                    <a
                        href="https://github.com/ML-Bioinfo-CEITEC/genomic_benchmarks"
                    >
                        ML-Bioinfo-CEITEC/genomic_benchmarks
                    </a>
                    benchmark
                </td>
            </tr>
            <tr>
                <td colspan="4">
                    load benchmark from HF
                    https://github.com/ML-Bioinfo-CEITEC/genomic_benchmarks/blob/main/notebooks/How_To_Use_Datasets_From_HF.ipynb
                </td>
            </tr>

            <tr>
                <td>DNABERT-2</td>
                <td>
                    <a href="https://magics.cs.northwestern.edu/index.html"
                        >MAGICS Labs</a
                    >, Northwestern University and Stony Brook University, USA
                </td>
                <td>
                    <a href="https://huggingface.co/zhihan1996/DNABERT-2-117M"
                        >zhihan1996/DNABERT-2-117M</a
                    >
                </td>
                <td>
                    <a href="https://github.com/MAGICS-LAB/DNABERT_2"
                        >MAGICS-LAB/DNABERT_2</a
                    >
                </td>
            </tr>
            <tr></tr>


            <tr>
              <td>nucleotide-transformer</td>
              <td>InstaDeep</td>
              <td>
                  <a
                      href="https://huggingface.co/collections/InstaDeepAI/nucleotide-transformer-65099cdde13ff96230f2e592"
                  >
                      InstaDeepAI/nucleotide-transformer
                  </a>
              </td>
              <td>
                <a
                  href="https://github.com/instadeepai/nucleotide-transformer">
                InstaDeepAI/nucleotide-transformer
                  </a>
              </td>
            </tr>
            <tr></tr>


            <tr>
              <td>GeneGPT</td>
              <td>NCBI, USA</td>
              <td>--</td>
              <td>
                <a href="https://github.com/ncbi/GeneGPT">ncbi/GeneGPT</a>
              </td>
            </tr>
            <tr>
              <td colspan="4">
                Natural language text / amino acids tool-use dataset
              </td>
            </tr>

            <tr>
                <td>ChatCell / Mol-Instructions</td>
                <td>
                    <a href="https://www.zjukg.org/">ZJUKG</a>, Zhejiang
                    University, China
                </td>
                <td>
                    <a href="https://huggingface.co/zjunlp/chatcell-large">
                        zjunlp/chatcell-large
                    </a>
                </td>
                <td>--</td>
            </tr>
            <tr>
                <td colspan="4">
                    ChatCell instruction dataset:
                    https://huggingface.co/datasets/zjunlp/ChatCell-Instructions <br/>
                    Mol-Instructions dataset:
                    https://huggingface.co/datasets/zjunlp/Mol-Instructions
                </td>
            </tr>

            <tr>
                <td>affinity_pred</td>
                <td>
                    Oak Ridge National Laboratory, USA
                </td>
                <td>
                    <a href="https://huggingface.co/models?search=jglaser%20affinity">
                        jglaser
                    </a>
                </td>
                <td>
                  <a href="https://github.com/ORNL/affinity_pred">
                    ORNL/affinity_pred
                  </a>
                </td>
            </tr>
            <tr>
            </tr>

            <tr>
                <td>ceLLama</td>
                <td><a href="https://celvox.co/">CelVox</a>, Netherlands</td>
                <td>-- (standard LLM with R packages)</td>
                <td>
                    <a href="https://github.com/CelVoxes/ceLLama">
                        CelVoxes/ceLLama
                    </a>
                </td>
            </tr>
            <tr>
                <td colspan="4">
                    Tutorial:
                    https://github.com/CelVoxes/ceLLama/blob/main/ceLLama/pbmc2700.ipynb
                </td>
            </tr>
        </table>
      </div>

        <h4>To be sorted</h4>
        <ul>
            <!--
https://www.biorxiv.org/content/10.1101/2024.11.26.625397v1 HyperMPNN
https://github.com/BioinfoMachineLearning/PoseBench

          https://www.nature.com/articles/s41586-024-07707-3
  https://blog.stephenturner.us/p/biorecap-r-package-for-summarizing-biorxiv-preprints-local-llm 
  https://opencitations.net/index/coci/api/v1#description 
https://github.com/a-r-j/graphein
https://github.com/lujiarui/esmdiff
 https://huggingface.co/collections/Esperanto/esperanto-fine-tuned-llm-based-models-673486672f0f0d8e0e14c985 (proteins, not just plants) 
 citation net around https://www.biorxiv.org/content/10.1101/2024.10.03.616542v1
and mde  https://github.com/Rhowell09/Machine-Learning-for-Rice-Nutritional-Components?tab=readme-ov-file 
add interplm,
 https://huggingface.co/datasets/tattabio/OG_prot90
  https://huggingface.co/seq-to-pheno
                https://huggingface.co/monsoon-nlp/tinyllama-mixpretrain-quinoa-sciphi
            </li> -->
            <li>
              https://huggingface.co/datasets/zwang09/protein_test_Llmma is a dataset of natural language descriptions
              and amino acids
            </li>
            <li>
              https://github.com/RosettaCommons/RFdiffusion/tree/main/examples
              is open source protein-folding
            </li>
            <li>
              LAMM: MIT Laboratory for Atomistic and Molecular Mechanics<br/>
              https://github.com/lamm-mit | https://huggingface.co/lamm-mit
            </li>
            <li>
                zxcyfr, shashwatsaini, AI4Protein, QizhiPei ,
                https://huggingface.co/collections/suke-sho/kmer-plant-bert-65e41ff19fb58a5115a291be
            </li>
        </ul>
    </body>
</html>
