<!doctype html>
<html>
    <head>
        <meta charset="utf-8" />
        <meta name="viewport" content="width=device-width, initial-scale=1" />
        <title>Plant-Based LLMs</title>
        <style type="text/css">
            html {
                font-family: sans-serif;
            }
            table {
                border: 1px solid #aaa;
                margin-right: 5px;
                width: calc(100% - 50px);
            }
            tr:nth-child(even) td {
                border-top: 1px solid #444;
            }
            tr:nth-child(odd) {
                background-color: #f0f0f0;
            }
            tr:nth-child(odd) td {
                border-bottom: 1px solid #aaa;
            }
            th,
            td {
                padding: 4px;
            }
            a {
                cursor: pointer;
                text-decoration: none;
            }
            a:hover {
                text-decoration: underline;
            }
            td[colspan='4'] a, td[colspan='5'] a {
              color: #0af;
              font-size: 85%;
            }
            .bracket {
              border: 1px solid #000;
              border-radius: 10px;
              display: flex;
              flex-direction: column;
              margin-bottom: 15px;
              padding: 10px;
              padding-right: 5px;
            }
            .bracket > div {
              display: flex;
              flex-direction: row;
            }
            .vineborder {
                border: 30px solid green;
                border-image: url('vine.jpg') 500 repeat;
                border-left-width: 0;
                width: 8px;
            }
            .plantless {
                border: 30px solid green;
                border-image: url('dna.png') 500 repeat;
                border-left-width: 0;
                width: 8px;
            }
        </style>
    </head>
    <body>
      <p>
        LLMs trained on plant genomes and proteomes.
        For context or questions, see
        <a href="https://github.com/mapmeld/plant-based-llms">the GitHub repo</a
        >.
      </p>

      <p>
        Skip to: <a href="#plant">Plant-Based</a> | <a href="#inclusive">Plant-Inclusive</a> | <a href="#other">Other Notable</a>
      </p>

      <a name="plant"></a>
      <div class="bracket">
        <h4>Plant-Based LLMs / Research</h4>
        <div>
            <table>
                <tr>
                    <th>Name</th>
                    <th>Source</th>
                    <th>Tokenization</th>
                    <th>HF Link</th>
                    <th>GH Link</th>
                </tr>
                <tr>
                    <td>AgroNT</td>
                    <td>InstaDeep</td>
                    <td>6bp/token, 1000x window</td>
                    <td>
                        <a
                            href="https://huggingface.co/InstaDeepAI/agro-nucleotide-transformer-1b"
                        >
                            InstaDeepAI/agro-nucleotide-transformer-1b
                        </a>
                    </td>
                    <td>
                        <a
                            href="https://github.com/instadeepai/nucleotide-transformer"
                        >
                            instadeepai/nucleotide-transformer
                        </a>
                    </td>
                </tr>
                <tr>
                    <td colspan="5">
                        <a href="https://github.com/instadeepai/nucleotide-transformer/blob/main/examples/inference.ipynb">Inference notebook</a>
                        -
                        <a href="https://github.com/instadeepai/nucleotide-transformer/blob/main/examples/inference_segment_nt.ipynb">Inference on segment</a>
                        -
                        For non-plant DNA, see:
                        <a href="https://huggingface.co/InstaDeepAI/nucleotide-transformer-2.5b-multi-species">nucleotide-transformer-2.5b-multi-species</a>
                    </td>
                </tr>

                <tr>
                    <td>PlantCaduceus</td>
                    <td>
                        <a href="https://kuleshov-group.github.io/"
                            >Kuleshov Group</a
                        >, Cornell University, USA
                    </td>
                    <td>1bp/token, 512 tokens x bidirectional</td>
                    <td>
                        <a
                            href="https://huggingface.co/kuleshov-group/PlantCaduceus_l32"
                        >
                            kuleshov-group/PlantCaduceus_l32
                        </a>
                    </td>
                    <td>
                        <a href="https://github.com/kuleshov-group/PlantCaduceus">
                            kuleshov-group/PlantCaduceus
                        </a>
                    </td>
                </tr>
                <tr>
                    <td colspan="5">
                        <a href="https://github.com/kuleshov-group/PlantCaduceus/blob/main/notebooks/examples.ipynb">Examples notebook</a>
                    </td>
                </tr>


                <tr>
                    <td>Plant-*-BPE / PDLLMs</td>
                    <td>
                        <a href="https://zhangtaolab.org/">Zhang Tao Lab</a>,
                        Yangzhou University, China
                    </td>
                    <td>nucleotide / BPE</td>
                    <td>
                        <a
                            href="https://huggingface.co/collections/zhangtaolab/plant-foundation-models-668514bd5c384b679f9bdf17"
                        >
                            collections/zhangtaolab/plant-foundation-models
                        </a>
                        (multiple architectures: Mamba, BERT, GPT)
                    </td>
                    <td>
                        <a href="https://github.com/zhangtaolab/plant_DNA_LLMs">
                            zhangtaolab/plant_DNA_LLMs
                        </a>
                    </td>
                </tr>
                <tr>
                    <td colspan="5">
                        <a href="https://github.com/zhangtaolab/Plant_DNA_LLMs/blob/main/scripts/run_finetune_test.sh">finetuning shell script</a>
                        -
                        <a href="https://github.com/zhangtaolab/Plant_DNA_LLMs/blob/main/scripts/run_inference_test.sh">inference shell script</a>
                    </td>
                </tr>

                <tr>
                    <td>PlantRNA-FM</td>
                    <td>
                    <a href="https://colalab.ai/">ColaLAB</a> @ University of Exeter and the
                    <a href="https://www.jic.ac.uk/about-us/">John Innes Centre</a> @ Norwich Research Park, UK</td>
                    <td>EsmTokenizer</td>
                    <td>
                        <a href="https://huggingface.co/yangheng/PlantRNA-FM">
                            yangheng/PlantRNA-FM</a
                        >
                    </td>
                    <td>
                        <a href="https://github.com/yangheng95/PlantRNA-FM"
                            >yangheng95/PlantRNA-FM</a
                        >
                    </td>
                </tr>
                <tr>
                    <td colspan="5">
                        <a href="https://github.com/yangheng95/PlantRNA-FM/blob/master/rice_region_classification.py">region classification in Python</a>
                        -
                        also see <a href="https://huggingface.co/yangheng/OmniGenome-52M">OmniGenome-52M model</a>
                        and
                        <a href="https://github.com/yangheng95/OmniGenBench">OmniGenBench benchmark for foundation models</a>
                    </td>
                </tr>

                <tr>
                    <td>PlantBERT</td>
                    <td>University of Potsdam, Germany</td>
                    <td>multiple nucleotides</td>
                    <td>
                        <a href="https://huggingface.co/nigelhartm/PlantBERT"
                            >nigelhartm/PlantBERT</a
                        >
                    </td>
                    <td>
                        <a href="https://github.com/nigelhartm/PlantBERT"
                            >nigelhartm/PlantBERT</a
                        >
                    </td>
                </tr>
                <tr>
                    <td colspan="5">
                        <a href="https://github.com/nigelhartm/PlantBERT/blob/main/scripts/08_finetune_all_gue.py">finetuning Python script</a>
                    </td>
                </tr>

                <tr>
                    <td>FloraBERT</td>
                    <td>
                        Institute for Applied Computational Sciences, Harvard
                        University, USA
                    </td>
                    <td>multiple nucleotides</td>
                    <td>
                        <a href="https://huggingface.co/Gurveer05/FloraBERT">
                            Gurveer05/FloraBERT
                        </a>
                    </td>
                    <td>
                        <a href="https://github.com/gurveervirk/florabert"
                            >gurveervirk/florabert</a
                        >
                    </td>
                </tr>
                <tr>
                    <td colspan="5">
                        <a href="https://github.com/benlevyx/florabert/blob/main/scripts/1-modeling/finetune.py">finetuning Python script</a>
                    </td>
                </tr>

                <tr>
                    <td>Farmer.chat</td>
                    <td>Digital Green and CGIAR, India</td>
                    <td>natural language LLMs</td>
                    <td>
                        <a href="https://huggingface.co/blog/digital-green-llm-judge">HF blog</a>
                    </td>
                    <td>
                        <a href="https://github.com/GooeyAI">GooeyAI</a>
                    </td>
                </tr>
                <tr>
                    <td colspan="5">
                    </td>
                </tr>


                <tr>
                    <td>1001G+ project</td>
                    <td>multiple, EU</td>
                    <td>pangenomic data</td>
                    <td>
                        --
                    </td>
                    <td>
                      <a href="https://1001genomes.org">
                        1001genomes.org
                      </a> from <i>Arabidopsis thaliana</i>
                    </td>
                </tr>
                <tr>
                </tr>

                <tr>
                    <td>protein-matryoshka-embeddings</td>
                    <td>it me, USA</td>
                    <td>amino acids</td>
                    <td>
                        <a
                            href="https://huggingface.co/monsoon-nlp/protein-matryoshka-embeddings"
                            >monsoon-nlp/protein-matryoshka-embeddings</a
                        >
                    </td>
                    <td>--</td>
                </tr>
                <tr>
                    <td colspan="5">
                        <a href="https://huggingface.co/datasets/monsoon-nlp/protein-pairs-uniprot-swissprot">dataset of pairs</a><br/>
                        <a href="https://huggingface.co/datasets/monsoon-nlp/greenbeing-proteins">plant protein and text descriptions</a><br/>
                        <a href="https://huggingface.co/datasets/monsoon-nlp/greenbeing-binary">plant protein descriptions converted to binary classification tasks</a>
                    </td>
                </tr>
                <tr>
                    <td>MoBiPlant</td>
                    <td>Universidad de Buenos Aires, Argentina</td>
                    <td>-</td>
                    <td>
                        <a href="https://huggingface.co/datasets/manufernandezbur/MoBiPlant">manufernandezbur/MoBiPlant</a>
                    </td>
                    <td>
                        <a href="https://github.com/manoloFer10/mobiplant">manoloFer10/mobiplant</a>
                    </td>
                </tr>
                <tr>
                    <td colspan="5">
                        This is a Q&A dataset for plant molecular biology. Claude and DeepSeek scored highly.
                    </td>
                </tr>
            </table>
            <div class="vineborder"></div>
        </div>
      </div>

        <a name="inclusive"></a>
        <div class="bracket">
        <h4>"Plant-Inclusive" LLMs / Research</h4>

        Trained on cells and proteins in general, including plants in training.
        <div>
            <table>
                <tr>
                <th>Name</th>
                <th>Source</th>
                <th>Tokenization</th>
                <th>HF Link</th>
                <th>GH Link</th>
                </tr>
                <tr>
                    <td>gLM2</td>
                    <td>
                        <a href="https://www.tatta.bio/">Tatta Bio</a>
                    </td>
                    <td>hybrid 1bp or 1 amino acid/token, 4096x window</td>
                    <td>
                        <a href="https://huggingface.co/tattabio/gLM2_650M">
                            tattabio/gLM2_650M
                        </a>
                    </td>
                    <td>
                        <a href="https://github.com/TattaBio/gLM2">
                            TattaBio/gLM2
                        </a>
                    </td>
                </tr>
                <tr>
                    <td colspan="5">
                        <a href="https://github.com/TattaBio/gLM2/blob/main/categorical_jacobian_gLM2.ipynb">example notebook</a> -
                        <a href="https://huggingface.co/tattabio/gLM2_650M_embed">gLM2's embeddings model</a>
                    </td>
                </tr>
                <tr>
                    <td>Evo 2</td>
                    <td>
                        Arc Institute and
                        <a href="https://evodesign.org/"
                            >Laboratory of Evolutionary Design</a
                        >, Stanford University, USA
                    </td>
                    <td>1bp/token, 1 million token context, StripedHyena bidirectional</td>
                    <td>
                        <a
                            href="https://huggingface.co/arcinstitute/evo2_40b"
                        >
                            arcinstitute/evo2_40b
                        </a>
                    </td>
                    <td>
                        <a href="https://github.com/arcinstitute/evo2">
                            arcinstitute/evo2
                        </a>
                        and
                        <a href="https://github.com/Zymrael/savanna"
                            >Zymrael/savanna</a
                        > for StripedHyena2 architecture
                    </td>
                </tr>
                <tr>
                    <td colspan="5">
                        Trained on <a href="https://huggingface.co/datasets/arcinstitute/opengenome2">opengenome2 dataset</a>
                        which includes multiple kingdoms of life, avoiding any viruses which affect eukaryotes.
                        <br/>
                        The previous Evo model was trained on prokaryote genomes only.<br/>
                        Independent researchers wrote <a href="https://www.biorxiv.org/content/10.1101/2025.09.10.675279v1">a paper about deploying Evo2</a><br/>
                        also read <a href="https://www.biorxiv.org/content/10.1101/2025.09.12.675911v1">generative design of bacteriophages</a> with Evo2.
                    </td>
                </tr>

                <tr>
                    <td>ZymCTRL</td>
                    <td><a href="https://www.aiproteindesign.com/">AI for Protein Design</a>,
                      Centre for Genomic Regulation, Spain
                    </td>
                    <td>amino acids</td>
                    <td>
                      <a href="https://huggingface.co/AI4PD/ZymCTRL">
                        AI4PD/ZymCTRL
                      </a>
                    </td>
                    <td>--</td>
                </tr>
                <tr>
                    <td colspan="5">
                      Enzyme design model<br/>
                        Improved on with reinforcement learning / DPO: <a href="https://github.com/AI4PDLab/DPO_pLM">code</a>
                    </td>
                </tr>


                <tr>
                    <td>ProtNote</td>
                    <td>Microsoft</td>
                    <td>amino acids + annotation codes</td>
                    <td>--</td>
                    <td>
                        <a href="https://github.com/microsoft/protnote">
                            microsoft/protnote
                        </a>
                    </td>
                </tr>
                <tr>
                    <td colspan="5">
                        <a href="https://github.com/microsoft/protnote/tree/main/notebooks">Multiple notebooks</a>
                    </td>
                </tr>

                <tr>
                    <td>ESM-2</td>
                    <td>Facebook / Meta</td>
                    <td>ESMTokenizer / amino acids</td>
                    <td>
                    <a href="https://huggingface.co/facebook/esm2_t48_15B_UR50D">
                        facebook/esm2_t48_15B_UR50D
                    </a>
                    </td>
                    <td>
                        <a href="https://github.com/facebookresearch/esm">
                            facebookresearch/esm
                        </a> (archived)
                    </td>
                </tr>
                <tr>
                    <td colspan="5">
                        Many finetuned models on HF, such as <a href="https://huggingface.co/blog/AmelieSchreiber/esmbind">AmelieSchreiber/esmbind</a>
                        <br/ >
                        <a href="https://github.com/pengzhangzhi/faesm">With "Flash Attention"</a>
                    </td>
                </tr>

                <tr>
                    <td>Bio Embeddings</td>
                    <td>Technical University of Munich, Germany</td>
                    <td>amino acids (?)</td>
                    <td>supports multiple models</td>
                    <td>
                        <a href="https://github.com/sacdallago/bio_embeddings"
                            >sacdallago/bio_embeddings</a
                        >
                    </td>
                </tr>
                <tr>
                <td colspan="5">
                    <a href="http://data.bioembeddings.com/">data.bioembeddings.com</a>
                </td>
                </tr>

                <tr>
                    <td>ProtTrans</td>
                    <td>multiple</td>
                    <td>amino acids</td>
                    <td><a href="https://huggingface.co/Rostlab">Rostlab</a></td>
                    <td>
                        <a href="https://github.com/agemagician/ProtTrans"
                            >agemagician/ProtTrans</a
                        >
                    </td>
                </tr>
                <tr></tr>

                <tr>
                    <td>GPN</td>
                    <td>
                        <a href="https://people.eecs.berkeley.edu/~yss/group.html"
                            >Song Lab</a
                        >, U.C. Berkeley, USA
                    </td>
                    <td>nucleotide</td>
                    <td>
                        <a
                            href="https://huggingface.co/collections/songlab/gpn-653191edcb0270ed05ad2c3e"
                        >
                            collections/songlab/gpn
                        </a>
                        (model, tokenizer, dataset)
                    </td>
                    <td>
                        <a href="https://github.com/songlab-cal/gpn">
                            songlab-cal/gpn
                        </a>
                    </td>
                </tr>
                <tr>
                    <td colspan="5">
                        <a href="https://github.com/songlab-cal/gpn/blob/main/examples/ss/basic_example.ipynb">Embedding notebook</a>
                        -
                        <a href="https://github.com/songlab-cal/gpn/blob/main/examples/msa/basic_example.ipynb">Multiple Sequence Alignment (MSA)</a>
                    </td>
                </tr>

                <tr>
                <td>BioM3</td>
                <td>U. Chicago, USA</td>
                <td>amino acid, protein design from natural text</td>
                <td>
                    --
                </td>
                <td>
                    --
                </td>
                </tr>
                <tr>
                <td colspan="5">
                    <a href="https://www.biorxiv.org/content/10.1101/2024.11.11.622734">BioM3 paper</a>
                </td>
                </tr>

                <tr>
                    <td>ProteinBERT</td>
                    <td>Hebrew University of Jerusalem, Israel</td>
                    <td>1/amino acid</td>
                    <td>
                        <a href="https://huggingface.co/GrimSqueaker/proteinBERT"
                            >GrimSqueaker/proteinBERT</a
                        >
                    </td>
                    <td>
                        <a href="https://github.com/nadavbra/protein_bert">
                            nadavbra/protein_bert
                        </a>
                    </td>
                </tr>
                <tr>
                    <td colspan="5">
                      <a href="https://github.com/nadavbra/protein_bert/blob/master/ProteinBERT%20demo.ipynb">demo notebook</a>
                    </td>
                </tr>

                <tr>
                    <td>Mistral-DNA</td>
                    <td>Paul Sabatier University, France</td>
                    <td>nucleotide</td>
                    <td>
                        <a
                            href="https://huggingface.co/RaphaelMourad/Mistral-DNA-v1-417M-Athaliana"
                        >
                            RaphaelMourad/Mistral-DNA-v1-417M-Athaliana
                        </a>
                    </td>
                    <td>
                        <a href="https://github.com/raphaelmourad/Mistral-DNA"
                            >raphaelmourad/Mistral-DNA</a
                        >
                    </td>
                </tr>
                <tr>
                    <td colspan="5">
                        <a href="https://github.com/raphaelmourad/Mistral-DNA/blob/main/scriptPython/script_finetune.ipynb">finetuning notebook</a>
                    </td>
                </tr>

                <tr>
                    <td>PAIR</td>
                    <td>Vector Institute / U. Toronto, Canada</td>
                    <td>amino acid</td>
                    <td>
                        <a href="https://huggingface.co/h4duan"> h4duan </a>
                        (finetunes of ESM, ProtT5)
                    </td>
                    <td>--</td>
                </tr>
                <tr>
                    <td colspan="5">
                        <a href="https://huggingface.co/h4duan/PAIR-prott5">Sample use in HF repo</a>
                    </td>
                </tr>

                <tr>
                    <td>ProtBFN</td>
                    <td>InstaDeep</td>
                    <td>amino acids</td>
                    <td>
                        <a
                            href="https://huggingface.co/InstaDeepAI/protein-sequence-bfn"
                        >
                            InstaDeepAI/protein-sequence-bfn
                        </a>
                    </td>
                    <td>
                        <a
                            href="https://github.com/instadeepai/protein-sequence-bfn"
                        >
                            instadeepai/protein-sequence-bfn
                        </a>
                    </td>
                </tr>
                <tr>
                    <td colspan="5">
                        <a href="https://github.com/instadeepai/protein-sequence-bfn/blob/main/sample.py">Sample Python script</a>
                    </td>
                </tr>

                <tr>
                    <td>CHEAP Embeddings</td>
                    <td>multiple, USA</td>
                    <td>amino acids</td>
                    <td>
                        <a href="https://huggingface.co/amyxlu/cheap-proteins"
                            >amyxlu/cheap-proteins</a
                        >, based on
                        <a href="https://huggingface.co/facebook/esmfold_v1"
                            >ESMFold</a
                        >
                    </td>
                    <td>
                        <a href="https://github.com/amyxlu/cheap-proteins">
                            amyxlu/cheap-proteins</a
                        >
                    </td>
                </tr>
                <tr></tr>

                <tr>
                    <td>ProTrek</td>
                    <td>
                        Westlake Representation Learning Lab, Westlake University,
                        China
                    </td>
                    <td>EsmTokenizer</td>
                    <td>
                        <a
                            href="https://huggingface.co/westlake-repl/ProTrek_650M_UniRef50"
                        >
                            westlake-repl/ProTrek_650M_UniRef50
                        </a>
                        and see
                        <a
                            href="https://huggingface.co/westlake-repl/SaProt_650M_AF2"
                            >SaProt_650M_AF2</a
                        >
                    </td>
                    <td>
                        <a href="https://github.com/westlake-repl/ProTrek"
                            >westlake-repl/ProTrek</a
                        >
                    </td>
                </tr>
                <tr>
                    <td colspan="5">
                       <a href="https://colab.research.google.com/drive/1On2xQU0d7351bIBgZpz2T0VUp2gZium0?usp=sharing">CoLab notebook</a>
                    </td>
                </tr>

                <tr>
                <td>Prot-xLSTM</td>
                <td>JKU Linz, Austria</td>
                <td>amino acids</td>
                <td><a href="https://ml.jku.at/research/Bio-xLSTM/downloads/Prot-xLSTM/">ml.jku.at</a></td>
                <td><a href="https://github.com/ml-jku/Prot-xLSTM">ml-jku/Prot-xLSTM</a></td>
                </tr>
                <tr>
                <td colspan="5">
                    <a href="https://github.com/ml-jku/Prot-xLSTM/tree/main/evaluation">evaluation examples</a> -
                    <a href="https://github.com/ml-jku/Prot-xLSTM/blob/main/examples/generation.ipynb">generation notebook</a> -
                    <a href="https://github.com/ml-jku/Prot-xLSTM/blob/main/examples/variant_fitness.ipynb">variant fitness notebook</a>
                </td>
                </tr>

                <tr>
                  <td>ProstT5</td>
                  <td>multiple</td>
                  <td>amino acid and 3Di structure tokens</td>
                  <td>
                    <a href="https://huggingface.co/Rostlab/ProstT5">
                      Rostlab/ProstT5
                    </a>
                  </td>
                  <td>
                    <a href="https://github.com/mheinzinger/ProstT5">
                      mheinzinger/ProstT5
                    </a>
                  </td>
                </tr>
                <tr>
                </tr>

                <tr>
                  <td>InterProt</td>
                  <td>multiple</td>
                  <td>ESM2 tokenizer</td>
                  <td>
                    <a href="https://huggingface.co/liambai/InterProt-ESM2-SAEs">
                      liambai/InterProt-ESM2-SAEs
                    </a>
                  </td>
                  <td>
                    <a href="https://github.com/etowahadams/interprot">
                      etowahadams/interprot
                    </a>
                  </td>
                </tr>
                <tr>
                </tr>

                <tr>
                    <td>RNA-FM / RhoFold</td>
                    <td>
                        multiple
                    </td>
                    <td>nucleotides</td>
                    <td>
                        (<a href="https://drive.google.com/drive/folders/1VGye74GnNXbUMKx6QYYectZrY7G2pQ_J">official</a> on Google Drive)<br/>
                        (<a href="https://huggingface.co/multimolecule/rnafm">unofficial</a> on HF/multimolecule)
                    </td>
                    <td>
                        <a href="https://github.com/ml4bio/RNA-FM"
                            >ml4bio/RNA-FM</a
                        >
                    </td>
                </tr>
                <tr></tr>

                <tr>
                    <td>Protein-Vec</td>
                    <td>
                        <a href="https://www.gleghornlab.com/">Gleghorn Lab</a>, U.
                        Delaware, USA
                    </td>
                    <td></td>
                    <td>
                        <a href="https://huggingface.co/lhallee/ProteinVec"
                            >lhallee/ProteinVec</a
                        >
                    </td>
                    <td>
                        <a href="https://github.com/lhallee/ProteinVecHuggingface"
                            >lhallee/ProteinVecHuggingface</a
                        >
                    </td>
                </tr>
                <tr></tr>

                <tr>
                    <td>MAMMAL / biomed.omics / biomed-multi-alignment</td>
                    <td>IBM, USA</td>
                    <td>amino acids, gene names</td>
                    <td>
                        <a
                            href="https://huggingface.co/ibm/biomed.omics.bl.sm.ma-ted-458m.protein_solubility"
                            >ibm/biomed.omics.bl.sm.ma-ted-458m.protein_solubility</a
                        >
                    </td>
                    <td>
                        <a
                            href="https://github.com/BiomedSciAI/biomed-multi-alignment"
                        >
                            BiomedSciAI/biomed-multi-alignment</a
                        >
                    </td>
                </tr>
                <tr>
                    <td colspan="5">
                        <a href="https://github.com/BiomedSciAI/biomed-multi-alignment/blob/main/tutorials/advanced_create_new_task.ipynb">
                          tutorial notebook creating a new task
                        </a>
                    </td>
                </tr>

                <tr>
                    <td>ProteinGLM</td>
                    <td>Tsinghua University, China</td>
                    <td>amino acids</td>
                    <td>
                        <a href="https://huggingface.co/Bo1015/proteinglm-100b-int4"
                            >Bo1015/proteinglm-100b-int4</a
                        >
                    </td>
                    <td>
                        <a href="https://github.com/allanchen95/xTrimoPGLM"
                            >allanchen95/xTrimoPGLM</a
                        >
                    </td>
                </tr>
                <tr>
                    <td colspan="5">
                       <a href="https://github.com/allanchen95/xTrimoPGLM/blob/main/finetune/lora_contact_prediction.py">
                         contact prediction Python script
                       </a>
                    </td>
                </tr>

                <tr>
                    <td>PSALM</td>
                    <td>Harvard University, USA</td>
                    <td>ESMTokenizer</td>
                    <td>
                        <a href="https://huggingface.co/ProteinSequenceAnnotation">ProteinSequenceAnnotation</a>
                    </td>
                    <td>
                        <a href="https://github.com/Protein-Sequence-Annotation/PSALM">
                          Protein-Sequence-Annotation/PSALM
                        </a>
                    </td>
                </tr>
                <tr>
                </tr>

                <tr>
                    <td>Protein-Llama-3-8B and Protein-Phi-3-mini</td>
                    <td>Esperanto Technologies, USA</td>
                    <td></td>
                    <td>
                        <a
                            href="https://huggingface.co/collections/Esperanto/esperanto-fine-tuned-llm-based-models-673486672f0f0d8e0e14c985"
                        >
                            collections/Esperanto/esperanto-fine-tuned-llm-based-models
                        </a>
                    </td>
                    <td>--</td>
                </tr>
                <tr>
                </tr>

                <tr>
                    <td>ProteinDT</td>
                    <td>multiple</td>
                    <td>amino acids</td>
                    <td>
                        <a
                            href="https://huggingface.co/chao1224/ProteinDT/tree/main/ProteinDT"
                        >
                            chao1224/ProteinDT
                        </a>
                    </td>
                    <td>
                    <a href="https://github.com/chao1224/ProteinDT">chao1224/ProteinDT</a>
                    </td>
                </tr>
                <tr>
                </tr>

                <tr>
                    <td>SeqDance</td>
                    <td><a href="http://www.columbia.edu/~ys2411/">Shen Lab</a>, Columbia University, USA</td>
                    <td>EsmTokenizer</td>
                    <td>
                        <a
                            href="https://zenodo.org/records/13909695"
                        >
                            on Zenodo
                        </a>
                    </td>
                    <td>
                    <a href="https://github.com/ShenLab/SeqDance">ShenLab/SeqDance</a>
                    </td>
                </tr>
                <tr>
                </tr>
            </table>
            <div class="vineborder"></div>
        </div>
      </div>

        <a name="other"></a>
        <div class="bracket">
        <h4>Other notable biology + LLM work</h4>
        <p>Not trained on any plants, but seems important to the bio research community. Includes datasets, tool-use</p>
        <div>
        <table>
            <tr>
                <th>Name</th>
                <th>Source</th>
                <th>HF Link</th>
                <th>GH Link</th>
            </tr>
            <tr>
                <td>AlphaGenome</td>
                <td>
                    Google DeepMind
                </td>
                <td>
                    -
                </td>
                <td>
                    <a href="https://github.com/google-deepmind/alphagenome">
                        google-deepmind/alphagenome
                    </a>
                </td>
            </tr>
            <tr>
                <td colspan="4">
                    <a href="https://deepmind.google/discover/blog/alphagenome-ai-for-better-understanding-the-genome/">blog</a>
                </td>
            </tr>
            <tr>
                <td>HyenaDNA</td>
                <td>
                    <a href="https://cs.stanford.edu/people/chrismre/"
                        >Hazy Research</a
                    >, Stanford University, USA
                </td>
                <td>
                    <a
                        href="https://huggingface.co/LongSafari/hyenadna-large-1m-seqlen"
                    >
                        LongSafari/hyenadna-large-1m-seqlen
                    </a>
                </td>
                <td>
                    <a href="https://github.com/HazyResearch/hyena-dna">
                        HazyResearch/hyena-dna
                    </a>
                </td>
            </tr>
            <tr>
                <td colspan="4">
                    <a href="https://github.com/HazyResearch/hyena-dna/tree/main/evals">evaluation scripts</a>
                </td>
            </tr>

            <tr>
                <td>Open MetaGenomic Dataset OMG</td>
                <td>
                    <a href="https://www.tatta.bio/">Tatta Bio</a>
                </td>
                <td>
                    <a href="https://huggingface.co/datasets/tattabio/OMG"
                        >datasets/tattabio/OMG</a
                    >
                </td>
                <td>
                    <a href="https://github.com/TattaBio/OMG"> TattaBio/OMG </a>
                </td>
            </tr>
            <tr></tr>

            <tr>
                <td>scGPT</td>
                <td>
                    <a href="https://wanglab.ai/">WangLab</a>, U. Toronto,
                    Canada
                </td>
                <td>--</td>
                <td>
                    <a href="https://github.com/bowang-lab/scGPT">
                        bowang-lab/scGPT
                    </a>
                </td>
            </tr>
            <tr>
                <td colspan="4">
                    <a href="https://github.com/bowang-lab/scGPT/blob/main/examples/finetune_integration.py">finetuning Python script</a>
                </td>
            </tr>

            <tr>
                <td>AllTheBacteria</td>
                <td>European Molecular Biology Laboratory, UK</td>
                <td>--</td>
                <td>
                    <a href="https://github.com/AllTheBacteria/AllTheBacteria">
                        AllTheBacteria/AllTheBacteria
                    </a>
                    dataset
                </td>
            </tr>
            <tr></tr>

            <tr>
                <td>GENA-LM</td>
                <td>
                  <a href="https://airi.net/">AIRI Institute</a>
                  , Russia
                </td>
                <td>
                  <a href="https://huggingface.co/collections/AIRI-Institute/dna-language-models-65b229c1cd9b73cf8462ac96">
                    AIRI-Institute/dna-language-models
                  </a>
                </td>
                <td>
                    <a href="https://github.com/AIRI-Institute/GENA_LM">
                        AIRI-Institute/GENA_LM
                    </a>
                    human DNA LLM
                </td>
            </tr>
            <tr></tr>

            <tr>
                <td>TYMEFLIES</td>
                <td>
                  Multiple, USA
                </td>
                <td>
                  --
                </td>
                <td>
                    <a href="https://www.nature.com/articles/s41564-024-01888-3">Nature paper</a>,
                    <i>Two decades of bacterial ecology and evolution in a freshwater lake</i> genomics,
                   <a href="https://blog.limnology.wisc.edu/2025/01/06/lake-loop-bacteria-in-lake-mendota-repeat-a-cycle-of-evolution-year-after-year/">wisc.edu article</a>,
                   <a href="https://bsky.app/profile/archaeal.bsky.social/post/3letjaxnhqc24">Bsky Post</a>
                </td>
            </tr>
            <tr></tr>

            <tr>
                <td>METAGENE</td>
                <td>
                  Multiple, USA
                </td>
                <td>
                   <a href="https://huggingface.co/metagene-ai/METAGENE-1">
                       metagene-ai/METAGENE-1
                   </a> pretrained on wastewater sample genomics
                </td>
                <td>
                    <a href="https://github.com/metagene-ai">
                       metagene-ai
                    </a>
                </td>
            </tr>
            <tr></tr>

            <tr>
                <td>Genomic Benchmarks</td>
                <td>CEITEC, Czech Republic</td>
                <td>
                    <a href="https://huggingface.co/katarinagresova"
                        >katarinagresova</a
                    >
                </td>
                <td>
                    <a
                        href="https://github.com/ML-Bioinfo-CEITEC/genomic_benchmarks"
                    >
                        ML-Bioinfo-CEITEC/genomic_benchmarks
                    </a>
                    benchmark
                </td>
            </tr>
            <tr>
                <td colspan="4">

                    <a href="https://github.com/ML-Bioinfo-CEITEC/genomic_benchmarks/blob/main/notebooks/How_To_Use_Datasets_From_HF.ipynb">Notebook loading benchmarks from HF</a>
                </td>
            </tr>

            <tr>
                <td>DNABERT-2</td>
                <td>
                    <a href="https://magics.cs.northwestern.edu/index.html"
                        >MAGICS Labs</a
                    >, Northwestern University and Stony Brook University, USA
                </td>
                <td>
                    <a href="https://huggingface.co/zhihan1996/DNABERT-2-117M"
                        >zhihan1996/DNABERT-2-117M</a
                    >
                </td>
                <td>
                    <a href="https://github.com/MAGICS-LAB/DNABERT_2"
                        >MAGICS-LAB/DNABERT_2</a
                    >
                </td>
            </tr>
            <tr></tr>


            <tr>
              <td>nucleotide-transformer</td>
              <td>InstaDeep</td>
              <td>
                  <a
                      href="https://huggingface.co/collections/InstaDeepAI/nucleotide-transformer-65099cdde13ff96230f2e592"
                  >
                      InstaDeepAI/nucleotide-transformer
                  </a>
              </td>
              <td>
                <a
                  href="https://github.com/instadeepai/nucleotide-transformer">
                InstaDeepAI/nucleotide-transformer
                  </a>
              </td>
            </tr>
            <tr>
                <td colspan="4">
                  Best finetuned performance on
                  <a href="https://github.com/kundajelab/DART-Eval">DART-Eval</a>
                </td>
            </tr>


            <tr>
              <td>GeneGPT</td>
              <td>NCBI, USA</td>
              <td>--</td>
              <td>
                <a href="https://github.com/ncbi/GeneGPT">ncbi/GeneGPT</a>
              </td>
            </tr>
            <tr>
              <td colspan="4">
                Natural language text / amino acids tool-use dataset
              </td>
            </tr>

            <tr>
                <td>ChatCell / Mol-Instructions</td>
                <td>
                    <a href="https://www.zjukg.org/">ZJUKG</a>, Zhejiang
                    University, China
                </td>
                <td>
                    <a href="https://huggingface.co/zjunlp/chatcell-large">
                        zjunlp/chatcell-large
                    </a>
                </td>
                <td>--</td>
            </tr>
            <tr>
                <td colspan="4">
                    <a href="https://huggingface.co/datasets/zjunlp/ChatCell-Instructions">ChatCell instruction dataset</a>
                    <br/>
                    <a href="https://huggingface.co/datasets/zjunlp/Mol-Instructions">Mol-Instructions dataset</a>
                </td>
            </tr>

            <tr>
                <td>affinity_pred</td>
                <td>
                    Oak Ridge National Laboratory, USA
                </td>
                <td>
                    <a href="https://huggingface.co/models?search=jglaser%20affinity">
                        jglaser
                    </a>
                </td>
                <td>
                  <a href="https://github.com/ORNL/affinity_pred">
                    ORNL/affinity_pred
                  </a>
                </td>
            </tr>
            <tr>
            </tr>

            <tr>
                <td>ceLLama</td>
                <td><a href="https://celvox.co/">CelVox</a>, Netherlands</td>
                <td>-- (standard LLM with R packages)</td>
                <td>
                    <a href="https://github.com/CelVoxes/ceLLama">
                        CelVoxes/ceLLama
                    </a>
                </td>
            </tr>
            <tr>
                <td colspan="4">
                    <a href="https://github.com/CelVoxes/ceLLama/blob/main/ceLLama/pbmc2700.ipynb">Tutorial notebook</a>
                </td>
            </tr>
        </table>
        <div class="plantless"></div>
        </div>
      </div>

        <h4>To be sorted</h4>
        <ul>
            <!--
https://www.biorxiv.org/content/10.1101/2024.11.26.625397v1 HyperMPNN
https://github.com/BioinfoMachineLearning/PoseBench

          https://www.nature.com/articles/s41586-024-07707-3
  https://blog.stephenturner.us/p/biorecap-r-package-for-summarizing-biorxiv-preprints-local-llm 
  https://opencitations.net/index/coci/api/v1#description 
https://github.com/a-r-j/graphein
https://github.com/lujiarui/esmdiff
 https://huggingface.co/collections/Esperanto/esperanto-fine-tuned-llm-based-models-673486672f0f0d8e0e14c985 (proteins, not just plants) 
 citation net around https://www.biorxiv.org/content/10.1101/2024.10.03.616542v1
and mde  https://github.com/Rhowell09/Machine-Learning-for-Rice-Nutritional-Components?tab=readme-ov-file 
add interplm,
 https://huggingface.co/datasets/tattabio/OG_prot90
  https://huggingface.co/seq-to-pheno
                https://huggingface.co/monsoon-nlp/tinyllama-mixpretrain-quinoa-sciphi
            </li> -->
            <li>
              https://huggingface.co/datasets/zwang09/protein_test_Llmma is a dataset of natural language descriptions
              and amino acids
            </li>
            <li>
                <a href="https://www.biorxiv.org/content/10.1101/2025.09.18.676967v1">
                 RFDiffusion3
                </a>
              is all-atom open source protein-folding
            </li>
            <li>
              LAMM: MIT Laboratory for Atomistic and Molecular Mechanics<br/>
              https://github.com/lamm-mit | https://huggingface.co/lamm-mit
            </li>
            <li>
                zxcyfr, shashwatsaini, AI4Protein, QizhiPei ,
                https://huggingface.co/collections/suke-sho/kmer-plant-bert-65e41ff19fb58a5115a291be
            </li>
        </ul>
        Vine border image produced by valadzionak_volha on Freepik
    </body>
</html>
